app/frontend/src/test/restMock.ts

```diff
@@ -0,0 +1,28 @@
+import { rest } from "msw";
+import { setupServer } from "msw/node";
+
+process.env = {
+  ...process.env,
+  REACT_APP_NOMINATIM_URL: "http://nominatim.test/",
+};
+
+const server = setupServer(
+  ...[
+    rest.get(
+      `${process.env.REACT_APP_NOMINATIM_URL!}search`,
+      (req, res, ctx) => {
+        return res(
+          ctx.json([
+            {
+              address: { city: "test city", country: "test country" },
+              lon: 1.0,
+              lat: 2.0,
+              display_name: "test city, test country",
+            },
+          ])
+        );
+      }
+    ),
+  ]
```

Comment:
```suggestion
  rest.get(
    `${process.env.REACT_APP_NOMINATIM_URL!}search`,
    (req, res, ctx) => {
      return res(
        ctx.json([
          {
            address: { city: "test city", country: "test country" },
            lon: 1.0,
            lat: 2.0,
            display_name: "test city, test country",
          },
        ])
      );
    }
  )
```
I think this also works without the extra spread